scipy/optimize/cython_optimize/examples/cython_optimize.ipynb

```diff
@@ -0,0 +1,1875 @@
+{
```

Comment:
Looking at this file, with its many lines of cython output, presumably dependent on the version of cython,I can't help thinking that this file doesn't below here but would serve interested users better in some other forum.